Mathlib/Geometry/Euclidean/MongePoint.lean

```diff
@@ -326,6 +326,17 @@ theorem orthocenter_eq_smul_vsub_vadd_circumcenter (t : Triangle ℝ P) :
   rw [orthocenter_eq_mongePoint, mongePoint_eq_smul_vsub_vadd_circumcenter]
   norm_num
 
+/-- **Sylvester's theorem** : The vector from the circumcenter to the orthocenter of a triangle
+is equal to the sum of the vectors from the circumcenter to each point. -/
+theorem orthocenter_vsub_circumcenter_eq_sum_vsub (t : Triangle ℝ P) :
```

Comment:
Is this true more generally as
```lean-4
theorem mongePoint_vsub_circumcenter_eq_sum_vsub {n} (t : Affine.Simplex ℝ P n) :
    t.mongePoint -ᵥ t.circumcenter = ∑ i, (t.points i -ᵥ t.circumcenter) := by
```

Reply:
No, you'd need to scale the RHS by `(n - 1)`.

Reply:
Ah nevermind, of course not.